### PR TITLE
Added Contact Page and did routing for the same

### PR DESCRIPTION
**Pull request by Priiyaa**
closes issue #18

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
-      <Home />
-      <Footer />
+      <Router>
+        <Routes>
+          {/* //* Home routes */}
+          <Route exact path="/" element={<HomePage />} />
+       
+          {/* //* Auth routes - USER*/}
+
+          <Route exact path="/user/register" element={<ContactForm />} />
```
Comment (anushbhatia) [inline src/App.js]: please use /contact

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Add padding inside the form placeholder elements
I would recommend using bootstrap elements for this so that responsiveness can be handeled too along side this

File: src/App.js (hunk 1 above)
Comment (Priiyaa) [reply src/App.js]: ok got it!

**Comment by Priiyaa** [comment]
### Added padding
**Mobile View**
contact mobile view
**Laptop and Tab view**

contact3

**Comment by Priiyaa** [comment]
I have changed the route to /user/contact

**Review by anushbhatia — CHANGES_REQUESTED** [review]
change the route path to /contact only

**Comment by Priiyaa** [comment]
> change the route path to /contact only

Done!

**Pull request merged**